Steepfile

```diff
@@ -0,0 +1,9 @@
+target :security do
+  signature "sig"
+
+  check "lib/datadog/security"
+
+  library "pathname", "set"
+  library "logger", "monitor"
+  library "tsort"
```

Comment:
These look like Ruby built-in gems. What's the guidance on listing them here? Do we have to list all the ones we use manually?

Reply:
Steep starts with only the core language signatures, you have to explicitly add elements of the standard library that are in use for the corresponding stdlib sigs to be loaded. I think the idea is that it caters for alternative implementations, eg. the mruby use case where the stdlib is much more modular.